ProjectShop/ProductApp/tests/factories.py

```diff
@@ -0,0 +1,83 @@
+import factory
+import random
+
+from ProductApp.models import *
```

Comment:
I believe importing an asterisk is not the best practice.

Reply:
thanks, fixed